src/customJs/socialShareTool/enableSocialProperty.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+const React = window.unlayer.React;
+
+const Toggle = ({ label, onChange }) => {
+  const [active, setActive] = React.useState(true);
+  const clickEvent = (e) => {
+    e.preventDefault();
```

Comment:
What are you preventing here?

Reply:
I did not understand, what is the default handler, I mean, what are you preventing?

Reply:
I correct, I need e.preventDefault to allow clicking on the toggle. The way more easily was attach the onclick Behavior over the toggle parent.

Reply:
I did not understand yet, we can clarify it later in a call. Thanks!